iconsets/affiliations/default/icondef.xml

```diff
@@ -5,7 +5,7 @@
 		<version>0.1</version>
 		<description>Adium Affiliations</description>
 		<creation>2010-06-10</creation>
-		<home>http://code.google.com/p/psi-dev</home>
+		<home>https://psi-im.org/</home>
```

Comment:
Why did you change Psi+ URL to Psi IM URL?

Reply:
Now it is in Psi code, you can look in README: https://github.com/psi-im/psi/blob/master/README.md

Reply:
They were created in Psi+ project and came from it. I think this link should point to Psi+ project for documenting of this fact.

@Ri0n What is your opinion of this?

Reply:
The question if we want to keep origin while the icons are maintained under Psi IM now.
From a user perspective to have a link to the resource where the icons are maintained would be better.